broadcasters/pygame_mixer_bc.py

```diff
@@ -0,0 +1,45 @@
+from loguru import logger
+from pygame import mixer
+
+from . import utils
+
+
+def check_if_available():
+    try:
+        mixer.init()
+        return True
+    except:
+        return False
+
+
+def start_processing_output(system_queue, pygame_mixer_q):
+    """Called by the broadcaster module to initialize a connection to a loudspeaker local to the system."""
+
+    def process():
+        # Go through and see what sounds we need to add
+        while True:
+            try:
+                for item in utils.get_all_from_queue(pygame_mixer_q):
+                    logger.debug("pygame_mixer: {}", item)
+                    if str(item).startswith("SOUND "):
```

Comment:
This sentinel value should be re-evaluated and a better protocol should be devised to ignore unwanted communication